src/tls13.c

```diff
@@ -5769,7 +5770,10 @@ int DoTls13ClientHello(WOLFSSL* ssl, const byte* input, word32* inOutIdx,
             ERROR_OUT(VERSION_ERROR, exit_dch);
         }
 
+        realMinor = ssl->version.minor;
+        ssl->version.minor = args->pv.minor;
         ret = HashInput(ssl, input + args->begin, helloSz);
+        ssl->version.minor = realMinor;
```

Comment:
Why?

Reply:
dtls 1.2 and dtls 1.3 differ in the way they compute the transcript hash (dtls1.3 does not include dtls-only fields).  we are downgrading to tls1.2 in that part of the code but `ssl->version.minor` can't be changed for the stateless constrain.